tests/testsuite.src/syn_misc.at

```diff
@@ -8200,3 +8200,82 @@ AT_CHECK([$COMPILE -freserved="XX*=BYTE-LENGTH" prog.cob], [0], [], [])
 AT_CHECK([$COBCRUN_DIRECT ./prog], [0], [1])
 
 AT_CLEANUP
+
+
+AT_SETUP([Condition IS ZERO AND])
+AT_KEYWORDS([bug 875])
+
+AT_DATA([prog.cob], [
+       IDENTIFICATION   DIVISION.
+       PROGRAM-ID.      prog.
+       DATA             DIVISION.
+       WORKING-STORAGE  SECTION.
+       01  V            PIC 9 VALUE 0.
+       01  W            PIC 9 VALUE 1.
+       PROCEDURE        DIVISION.
+           IF V IS ZERO
+              DISPLAY "V IS ZERO"
+           END-IF.
+           IF V IS ZERO AND W EQUAL 1
+              DISPLAY "V IS ZERO AND W EQUAL 1"
+           END-IF.
+           IF W EQUAL 1 AND V IS ZERO
+              DISPLAY "W EQUAL 1 AND V IS ZERO"
+           END-IF.
+           IF W IS POSITIVE
+              DISPLAY "W IS POSITIVE"
+           END-IF.
+           IF W IS NEGATIVE
+              DISPLAY "W IS NEGATIVE"
+           END-IF.
+           IF W IS POSITIVE AND V EQUAL 0
+              DISPLAY "W IS POSITIVE AND V EQUAL 0"
+           END-IF.
+           IF V EQUAL 0 AND W IS POSITIVE
+              DISPLAY "V EQUAL 0 AND W IS POSITIVE"
+           END-IF.
+])
+
+AT_CHECK([$COMPILE prog.cob], [0], [],
+[prog.cob:21: warning: unsigned 'W' may not be LESS THAN ZERO
+])
+
+AT_CHECK([./prog], [0], [V IS ZERO
+V IS ZERO AND W EQUAL 1
+W EQUAL 1 AND V IS ZERO
+W IS POSITIVE
+W IS POSITIVE AND V EQUAL 0
+V EQUAL 0 AND W IS POSITIVE
+], [])
+
+AT_CLEANUP
+
+AT_SETUP([abbreviated conditions with multiple words operators])
+AT_KEYWORDS([bug 880])
+
+AT_DATA([prog.cob], [
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID.    CHECKBOOL.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+       01  VAR1                    PIC X(16) VALUE "#0001".
+       01  VAR2                    PIC X(16) VALUE "#0002".
+       01  VAR3                    PIC X(16) VALUE "#0003".
+       01  VAR4                    PIC X(16) VALUE "#0004".
+       PROCEDURE DIVISION.
+       MAIN-PROGRAM SECTION.
+       INIZIO.
+           IF VAR1 = (VAR2 AND VAR3 AND VAR4)
+              DISPLAY "TRUE" UPON STDERR
+	   END-IF
+           IF VAR1 NOT = (VAR2 AND VAR3 AND VAR4)
+              DISPLAY "TRUE" UPON STDERR
+           END-IF
+           IF VAR1 NOT > (VAR2 AND VAR3 AND VAR4)
+              DISPLAY "TRUE" UPON STDERR
+           END-IF
+           GOBACK.
```

Comment:
Please adjust the test (which also gets rid of a leading tab):

```cob
           IF VAR1 = (VAR2 AND VAR3 AND VAR4)
              DISPLAY "TRUE 1".
           IF VAR1 NOT = (VAR2 AND VAR3 AND VAR4)
              DISPLAY "TRUE 2".
           IF VAR1 NOT > (VAR2 AND VAR3 AND VAR4)
              DISPLAY "TRUE 3".
```

then move both tests from syn_misc.at to run_fundamental.at and also check the second output, which - according to Micro Focus - should be 
> TRUE 2
> TRUE 3